extra-python/pymediainfo/spec

```diff
@@ -0,0 +1,3 @@
+VER=5.0.3
+SRCS="https://files.pythonhosted.org/packages/1f/ec/da04fb8cbf5df9684b549bc3cb5c2010a0661da4bc626a6d531462222511/pymediainfo-${VER}.tar.gz"
```

Comment:
You should use `pypi.io` instead. See `extra-python/april` to see how these links work.